src/Commands/Backup/Schedule/CancelCommand.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Backup\Schedule;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+
+/**
+ * Class CancelCommand
+ * @package Pantheon\Terminus\Commands\Backup\Schedule
+ */
+class CancelCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+
+    /**
+     * Cancels a regular backup schedule
```

Comment:
You're gonna kill me, but I realized that this command is probably too prosaic all around. It really should be "holy crap, you're disabling automatic backups" not "cancel backup schedule".